### ISSUE TITLE
can't achieve the reported accuracy on VGG19

### ISSUE THREAD

**Issue by zeal-github** [kuangliu/pytorch-cifar#55]
Have anyone train a VGG19 network used this code? I used the total exactly the same code in the repository except using a lr_schuduler to change the learning rate automatically. But I can only achieve the accuracy around 88%. Can anyone tell me what's wrong?

**Comment by EnJiang** [comment]
Did you train use smaller lr as the author said? For me it would achieve ~90.8%, but I still cannot get the reported 92.64%.
@kuangliu care to share your idea?

**Comment by hjinlee88** [comment]
I've got 93.37% with ubuntu 16.04, cuda 9.2, cudnn 7.2, pytorch 0.4.1, and 4 * gtx 1080 ti.
I use 
batch_size=128 (means 32 for each gpu) 
epoch=450 
learning rate schedule 
    0.1 when epoch < 150, 
    0.01 when epoch < 300, 
    0.001 when epoch < 450
momentum=0.9, weight_decay=5e-4

**Comment by gaosh** [comment]
@zeal-github @EnJiang I think the main problem is that you didn't set the learning rate schedule right. I got 93.69% using the same learning rate schedule as default. You can see pull request #26 for a automatic learning rate schedule.

**Comment by myhub** [comment]
> @hjinlee88 @gaosh Are you sure your results reproducible? I used the same learning rate schedule with yours, but none of my experiments could reach 92% acc. I especially have done many experiments on vgg16, the best result is 90.68%, which is much lower than 92.64% reported by the author.

I think We should use The same random seed, or IT  is not reproducible. any body share The random seed

**Comment by gaosh** [comment]
@myhub @lhyfst if the base result is 90.68%, I don't think change random seed would help. The margin is near 3%, such huge difference can't be solely explained by random seed.

**Comment by atulshanbhag** [comment]
What training error is everyone achieving?

**Comment by cbasemaster** [comment]
is it training accuracy or testing accuracy?

**Comment by iversonicter** [comment]
@cbasemaster , It's testing accuracy. Training accuracy is meaningless. I follow the hjinlee88 suggestion, the best test accuracy is 92.75%.

**Comment by samuela** [comment]
I'm not seeing VGG19 listed anywhere on the [README](https://github.com/kuangliu/pytorch-cifar/blob/49b7aa97b0c12fe0d4054e670403a16b6b834ddd/README.md), only VGG16. Are you guys testing on VGG16 or VGG19?
